ppm-main/src/main/java/com/ctu/se/oda/model11/api/MaterialApi.java

```diff
@@ -0,0 +1,68 @@
+package com.ctu.se.oda.model11.api;
+
+import com.ctu.se.oda.model11.interfaces.IMaterialApplication;
+import com.ctu.se.oda.model11.mappers.IMainMapper;
+import com.ctu.se.oda.model11.models.commands.requests.material.CreateMaterialCommandRequest;
+import com.ctu.se.oda.model11.models.commands.requests.material.UpdateMaterialCommandRequest;
+import com.ctu.se.oda.model11.models.commands.responses.material.CreateMaterialCommandResponse;
+import com.ctu.se.oda.model11.models.commands.responses.material.UpdateMaterialCommandResponse;
+import com.ctu.se.oda.model11.models.material.CreateMaterialRequest;
+import com.ctu.se.oda.model11.models.material.UpdateMaterialRequest;
+import com.ctu.se.oda.model11.models.queries.responses.material.RetrieveMaterialQueryResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/materials")
+public class MaterialApi {
+    @Autowired
+    private IMainMapper<CreateMaterialRequest, CreateMaterialCommandRequest> createMaterialMapper;
+    @Autowired
+    private IMainMapper<UpdateMaterialRequest, UpdateMaterialCommandRequest> updateMaterialMapper;
+    @Autowired
+    private IMaterialApplication materialApplication;
+    @PostMapping("")
+    public ResponseEntity<CreateMaterialCommandResponse> createMaterial(@RequestBody CreateMaterialRequest createMaterialRequest) {
+        return new ResponseEntity<>(
+                this.materialApplication.createMaterial(this.createMaterialMapper.convert(createMaterialRequest)),
+                HttpStatus.CREATED
+        );
+    }
+    @PutMapping("/{materialId}")
+    public ResponseEntity<UpdateMaterialCommandResponse> updateMaterial(@RequestBody UpdateMaterialRequest updateMaterialRequest, @PathVariable String materialId) {
+        return new ResponseEntity<>(
+                this.materialApplication.updateMaterial(this.updateMaterialMapper.convert(updateMaterialRequest), Long.valueOf(materialId)),
+                HttpStatus.OK
+        );
+    }
+    @GetMapping("")
```

Comment:
Remove the empty quotation mark **""** for better readability.
`@GetMapping("") -> @GetMapping()`